Snowflake.Data.Tests/FileUploadDownloadLargeFilesIT.cs

```diff
@@ -0,0 +1,134 @@
+/*
+ * Copyright (c) 2023 Snowflake Computing Inc. All rights reserved.
+ */
+
+using System;
+using System.IO;
+using System.IO.Compression;
+using System.Security.Cryptography;
+using NUnit.Framework;
+using Snowflake.Data.Client;
+
+namespace Snowflake.Data.Tests
+{
+
+    [TestFixture]
+    public class FileUploadDownloadLargeFilesIT : SFBaseTest
```

Comment:
The test does almost the same as tests in SFPutGetTest.
Maybe it should be a part of SFPutGetTest? Then it will be more consistent.

Reply:
SFPutGetTest is a test suite which (in setup for each test) creates a table and a stage. For my purpose I don't want to create any tables. I would like just to verify if uploading/downloading works for large files. So it is in a separate class because the test has a totally different setup.